pkgs/applications/misc/tandoor-recipes/default.nix

```diff
@@ -131,6 +134,10 @@ python.pkgs.pythonPackages.buildPythonPackage rec {
     runHook postInstall
   '';
 
+  postInstall = ''
+    wrapProgram $out/lib/tandoor-recipes/manage.py --prefix PYTHONPATH : "$PYTHONPATH"
+  '';
```

Comment:
This is already done in `installPhase`, see line 128 on this commit.

Reply:
I applied this change because otherwise `nix-build -A tandoor-recipes.tests.runManagePy` *fails.*